src/org/usfirst/frc/team2473/robot/subsystems/DrivePower.java

```diff
@@ -0,0 +1,197 @@
+package org.usfirst.frc.team2473.robot.subsystems;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+
+/*
+ * Instead of always inputting doubles for motor powers, a DrivePower object can be passed
+ * to multiply drive-tuning constants automatically.
+ */
+public class DrivePower {
```

Comment:
Implement drive power in the Drive Subsystem instead

Reply:
Agreed